src/YieldDistributor.sol

```diff
@@ -63,6 +63,8 @@ contract YieldDistributor is OwnableUpgradeable {
     uint256 public lastClaimedBlockNumber;
     // @notice The total number of votes cast in the current cycle
     uint256 public currentVotes;
+    // @notice How much of the yield is divided equally among projects
```

Comment:
this is not upgrade safe. it will shift the storage location of everything below it. new variables need to be appended to the end of the storage layout. so underneath all existing declarations.

Reply:
```
    └─ ← [Revert] revert: Upgrade safety validation failed:
✘  src/YieldDistributor.sol:YieldDistributor (upgrades from test/upgrades/v1.0.0/YieldDistributor.sol:YieldDistributor)

      src/YieldDistributor.sol:67: Inserted `yieldFixedSplit`
        > New variables should be placed after all existing inherited variables

FAILED
```

Just merged and checked, it does fail!

Reply:
addressed with 6b024f3